upgrade_test.py

```diff
@@ -337,35 +337,49 @@ def test_rolling_upgrade(self):
         # wait for the 20m read workload to finish
         self.verify_stress_thread(read_20m_cs_thread_pool)
 
+        self.verify_stress_thread(entire_write_cs_thread_pool)
+
+        verify_stress_after_cluster_upgrade = self.params.get('verify_stress_after_cluster_upgrade')
+        verify_stress_cs_thread_pool = self.run_stress_thread(stress_cmd=verify_stress_after_cluster_upgrade)
+        self.verify_stress_thread(verify_stress_cs_thread_pool)
+
         self.log.info('Run some Queries to verify data AFTER UPGRADE')
         self.verify_db_data()
 
-        # sst3 workload: verify data by simple read cl=ALL
-        self.log.info('Starting c-s sst3 workload to verify data by simple read')
-        stress_cmd_sst3_verify_read = self.params.get('stress_cmd_sst3_verify_read')
-        sst3_cs_thread_pool = self.run_stress_thread(stress_cmd=stress_cmd_sst3_verify_read, profile='data_dir/sst3_schema.yaml')
+        # complex workload: verify data by simple read cl=ALL
+        self.log.info('Starting c-s complex workload to verify data by simple read')
+        stress_cmd_complex_verify_read = self.params.get('stress_cmd_complex_verify_read')
+        complex_cs_thread_pool = self.run_stress_thread(stress_cmd=stress_cmd_complex_verify_read, profile='data_dir/complex_schema.yaml')
+        # wait for the read complex workload to finish
+        self.verify_stress_thread(complex_cs_thread_pool)
+
+        ## complex workload: verify data by multiple ops
```

Comment:
@amoskong add yourself a task to review these steps separately.
How many minutes they add to the test and what is the extra value they bring to the test.
Then, we can decide if to remove them from the code or not.

Reply:
OK